common.mk

```diff
@@ -12,15 +12,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Common path
+COMMON_PATH := device/sony/common
```

Comment:
This one might have been a conflict resolution mistake? 008f9c214db1dc24a49587f9400f75c913da625e doesn't have it :)

Reply:
> This one might have been a conflict resolution mistake? [008f9c2](https://github.com/sonyxperiadev/device-sony-common/commit/008f9c214db1dc24a49587f9400f75c913da625e) doesn't have it :)

Yup, this is the result after solving cherry-pick conflict. For Q we only need to define sm8150.

Reply:
Yes, don't forget to remove it then :)